### PR TITLE
Chore: merge from ce11411 to f7426c0

### PR DESCRIPTION
**Pull request by julien-deramond**
- [x] https://github.com/twbs/bootstrap/commit/ce114117a8710016022c40e717cb1f3ec95a5d7f
- [x] ~~https://github.com/twbs/bootstrap/commit/d58e86e7308e82d9564f19577adbc5b228f06d58~~: we dont' have table head/foot in Boosted. We might need to reintroduce them in a separate PR.
  - [x] Create https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/2337 to reintegrate them
- [x] https://github.com/twbs/bootstrap/commit/dac60074dc6cdc6c67a9e97fe46f4f43366df839
- [x] https://github.com/twbs/bootstrap/commit/f7426c08e7818ff3eb77f7c7cde54ca6d28009b2

### CODE REVIEW

File: js/tests/visual/modal.html [hunk 1]
```diff
@@ -13,21 +13,25 @@
...
-            <a class="nav-link" href="#">Link</a>
-          </li>
-        </ul>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+      <div class="container-fluid">
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarResponsive">
+          <a class="navbar-brand text-white" href="#">This shouldn't jump!</a>
```
Comment (julien-deramond) [inline js/tests/visual/modal.html:22]: The rendering is not ODS compatible but it's just for some tests related to the fact that the navbar jumps or not. So IMO nevermind. I've just added a `.text-white` just to force it to be visible.

File: site/content/docs/5.3/components/orange-navbar.md [hunk 2]
```diff
@@ -192,7 +192,7 @@ An additional navbar (with text or icon items) can be added on the right of the
               </svg>
               <span class="visually-hidden">Basket</span>
               <span class="position-relative align-self-start">
-                <span class="badge bg-info rounded-pill position-absolute top-0 fs-6 text-white translate-middle">
+                <span class="badge text-bg-info rounded-pill position-absolute top-0 fs-6 text-white translate-middle">
```
Comment (julien-deramond) [inline site/content/docs/5.3/components/orange-navbar.md:195]: There's no impact here but we will need to define in the dark mode implementation whether `.text-bg-info` should render the text color in white automatically or black. For now, `.text-white` is still needed.

File: site/content/docs/5.3/components/badge.md
Comment (louismaximepiton) [inline site/content/docs/5.3/components/badge.md:1]: Missing one below, the basket example.

File: js/tests/visual/modal.html (hunk 1 above)
Comment (louismaximepiton) [reply js/tests/visual/modal.html:22]: Yeah we changed the markup a bit in Boosted So I'm totally fine dealing with it here.

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
One comment to tackle and we're good to go

File: site/content/docs/5.3/components/badge.md
Comment (julien-deramond) [reply site/content/docs/5.3/components/badge.md:1]: Good catch. Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2336/commits/e744afa9c30cb209ef8fd1709c5f29f88fb65d8a.

**Review by louismaximepiton — APPROVED** [review]

**Pull request merged**